src/tests/multihost/admultidomain/readme.rst

```diff
@@ -0,0 +1,133 @@
+AD Provider Test Suite
+======================
+
+This directory contains test automation for SSSD AD Provider.
```

Comment:
Is this directory going to be used as the model for all AD provider tests or mostly just multidomain?  If the latter, we should clarify that in the readme I think.

Reply:
Updated and removed unused function documentation.